### PR TITLE
Makes the integration work out of the box

### PR DESCRIPTION
**Pull request by bsneed**
- Fixes issue w/ supported OS version.
- Fixes framework inclusions.
- Adds lines to actually insert source_files.
- Adds lines to fix lack of arm64 slide in Nielsen binary.
- Adds Nielsen as an actual dependency of this pod.

### CODE REVIEW

File: Example/Segment-Nielsen-DCR/SEGNielsenDCRAppDelegate.m
Comment (briemcnally) [inline Example/Segment-Nielsen-DCR/SEGNielsenDCRAppDelegate.m:12]: Is this an umbrella header? Curious why naming changed?

Comment (bsneed) [reply Example/Segment-Nielsen-DCR/SEGNielsenDCRAppDelegate.m:12]: No, it's just a way to import a framework without having to get into header specifics.  Due to our naming, -'s are converted to _ and so this just makes it easier because we really don't care, we just need access to stuff in the framework.

File: Segment-Nielsen-DCR.podspec
Comment (briemcnally) [inline Segment-Nielsen-DCR.podspec:24]: Mind explaining what exactly this is doing?

Comment (bsneed) [reply Segment-Nielsen-DCR.podspec:24]: Since Nielsen's SDK does include an arm64 slice (needed for arm based Macs coming soon), we're excluding it from our library as well so things will build correctly.

**Comment by briemcnally** [comment]
LGTM other than just clarification on one comment and we should fix the build issue with the CircleCI before merging.

**Review by briemcnally — APPROVED** [review]
Approving so can be deployed once CI is fixed. Thanks for digging in @bsneed

**Pull request merged**
